packages/core/src/backend.ts

```diff
@@ -749,8 +750,8 @@ export class Backend<EF extends BaseField = UnknownField, BC extends BackendClas
     return entryValue;
   }
 
-  getMedia(folder?: string | undefined, mediaPath?: string | undefined) {
-    return this.implementation.getMedia(folder, mediaPath);
+  getMedia(folder?: string | undefined, mediaPath?: string | undefined, folderSupport?: boolean) {
+    return this.implementation.getMedia(folder, mediaPath, folderSupport);
```

Comment:
Since only the proxy backend uses `mediaPath` / `publicFolder`, I would prefer we reorder the parameters to have public folder at the end. Then the backends that don't need it don't have to include it.
```suggestion
    return this.implementation.getMedia(folder, folderSupport, mediaPath);
```

Reply:
I changed it everywhere. As I already had to touch every backend, that wasn't a problem.